Update instructions on using prettier-plugin-astro (#8563)
withastro/docs: src/content/docs/en/editor-setup.mdx

To add support for formatting `.astro` files outside of the editor (e.g. CLI) or inside editors that don't support our editor tooling, install [the official Astro Prettier plugin](https://github.com/withastro/prettier-plugin-astro).

<Steps>
1. Install `prettier` and `prettier-plugin-astro`.

    <PackageManagerTabs>
      <Fragment slot="npm">
      ```shell
      npm install --save-dev prettier prettier-plugin-astro
      ```
      </Fragment>
      <Fragment slot="pnpm">
      ```shell
      pnpm add -D prettier prettier-plugin-astro
      ```
      </Fragment>
      <Fragment slot="yarn">
      ```shell
      yarn add --dev prettier prettier-plugin-astro
      ```
      </Fragment>
    </PackageManagerTabs>

> 2. Create a `.prettierrc.mjs` config file and add `prettier-plugin-astro` to it.

Reviewer: ```suggestion
2. Create a `.prettierrc.mjs` config file at the root of your project and add `prettier-plugin-astro` to it.
```
I'm assuming that's where it goes? Let's say where to put the file!

>     It's also recommended to manually specify the parser being used for Astro files.

Reviewer: ```suggestion
    In this file, also manually specify the parser for Astro files.
```
If it's recommended, then let's just tell them to do it!

    ```js title=".prettierrc.mjs"
    /** @type {import("prettier").Config} */
    export default {
      plugins: ['prettier-plugin-astro'],
      overrides: [
        {
          files: '*.astro',
          options: {
            parser: 'astro',
          },
        },
      ],
    };
    ```

> 3. Use `prettier . --write` to format your files.

Reviewer: ```suggestion
3. Run the command `prettier . --write` in the terminal to format your files.
```
